src/modules/dashboard/components/ActionsPage/ActionsPage.css

```diff
@@ -21,7 +33,19 @@
    */
   margin: 0 13.19% 0 24.3%;
   width: 900px;
-  border-top: 1px solid color-mod(var(--temp-grey-blue-7) alpha(15%));
+}
+
+.dividerTop {
+  composes: setMarginWidth;
+  height: 1px;
+  background-color: color-mod(var(--temp-grey-blue-7) alpha(15%));
+}
```

Comment:
Usually, when I do dividers, I just go without a border, and just set a `background-color`, and a `height` of `1px`.

But this is just a side note, you can do it however you like it :)